v1/company_ids.json

```diff
@@ -577,7 +577,6 @@
     { "code": 575, "name": "WaveWare Technologies Inc." },
     { "code": 576, "name": "Argenox Technologies" },
     { "code": 577, "name": "Bragi GmbH" },
-    { "code": 578, "name": "16Lab Inc" },
```

Comment:
I think this needs to be undone for the checks to continue. Might need to revisit the script so as to not remove Company IDs.